src/index.ts

```diff
@@ -77,7 +80,7 @@ module.exports = (options = {}) => {
     server,
     webhook,
     receive,
-    logger,
+    logger: Logger,
```

Comment:
change this back to `logger,`, you want to export the probot logger, not bunyan itself